spec.md

```diff
@@ -70,12 +70,11 @@ system. This is most typically when that system receives an external signal
 (e.g. an IoT sensor or period of inactivity).
 
 #### Event
-Data representing an occurrence, a change in state, that something happened
-(or did not happen).  Events include context and data.  Each occurrence MAY be
-uniquely identified with data in the event. Events ought not to be confused
-with messages which are used to transport or distribute data without assumptions
-regarding its semantic. Events are considered to be facts that have no given
-destination. Events are used to notify other systems that something has happened.
+An "event" is a data record expressing an occurrence and its context. Events
+are routed from the emitting source to interested parties for the purpose of
+notifying them about the source occurrence. The routing can be performed based
+on information contained in the event, but an event will generally not identify
```

Comment:
an event will not identify a specific routing destination.

The goal is to de-couple source and action, so I think we need to remove "generally"

Reply:
@clemensv I think you agreed to remove the word "generally".

Reply:
@clemensv if you make this minor tweak we might be able to merge tomorrow

Reply:
@djrosanova with @clemensv out are you ok with me removing the word "generally" for him? I believe @clemensv agreed to it (I just can't remember where :-) ). I can also do the rebase if so.